### PR TITLE
Add info about 'config/render/capm3.yaml in README 

### PR DESCRIPTION
**Pull request by kashifest**
This PR adds important information about 'config/render/capm3.yaml in README. This was not clear before. Please check the README for more detail

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -27,3 +27,13 @@ components, see the [Metal³ docs](https://github.com/metal3-io/metal3-docs).
 * [Configuration](docs/configuration.md)
 * [Testing](docs/testing.md)
 * [Publishing Images](docs/publishing-images.md)
+
+### Important Notes
+
+Whenever there is a change in `config\` directory, please remember to run the
```
Comment (furkatgofurov7) [inline README.md:33]: `config/` maybe?

**Review by furkatgofurov7 — COMMENTED** [review]
Looks good to me, small nit inline

File: README.md (hunk 1 above)
Comment (kashifest) [reply README.md]: Done. Thanks

**Comment by fmuyassarov** [comment]
LGTM

File: README.md [hunk 2]
```diff
@@ -27,3 +27,13 @@ components, see the [Metal³ docs](https://github.com/metal3-io/metal3-docs).
...
+
+### Important Notes
+
+Whenever there is a change in `config/` directory, please remember to run the
+following command:
+
+`make manifests`
+
+This will render the `config/render/capm3.yaml`. Please do not change the
+content of `config/render/capm3.yaml` manually.
```
Comment (dhellmann) [inline README.md:39]: Does the `generate` CI job accurately test to ensure this is done? It looks like it checksums the files under `config/` then runs `make generate manifests` and then compares the new checksums with the old. Is it capturing all of the right files?

Comment (dhellmann) [reply README.md:39]: https://prow.apps.ci.metal3.io/view/gcs/metal3-prow-artifacts/pr-logs/pull/metal3-io_baremetal-operator/702/generate/1321414863867613184 shows the job output for this PR

Comment (kashifest) [reply README.md:39]: Yes does. Its part from the `manifests` make target and as you pointed out it does the `make generate manifests` . One of the PRs tried to modify the `config/render/capm3.yaml` unknowingly. It was caught by generate, [here](https://storage.googleapis.com/metal3-prow-artifacts/pr-logs/pull/metal3-io_baremetal-operator/700/generate/1321372458866970627/build-log.txt) is the output of the failed run

Comment (dhellmann) [reply README.md:39]: OK, great. I wanted to make sure we weren't just relying on the docs to catch this sort of mistake.

**Pull request merged**